tests/test_lights.py

```diff
@@ -49,9 +49,9 @@ def test_init():
     assert isinstance(light, pyvista.Light)
     assert light.position == position
     assert light.focal_point == focal_point
-    assert light.ambient_color == color
-    assert light.diffuse_color == color
-    assert light.specular_color == color
+    assert light.ambient_color == pyvista.Color(color)
+    assert light.diffuse_color == pyvista.Color(color)
+    assert light.specular_color == pyvista.Color(color)
```

Comment:
They could, but only for very specific values of `color` (e.g. `(0.2, 0.2, 0.2)` would work but `(0.1, 0.1, 0.1)` wouldn't) because of the internal integer representation of Colors. The robuster check is to compare `Color` objects instead of their `f_rgba` representations.

Reply:
Very good point, thanks.